src/Integration/TransactionIntegration.php

```diff
@@ -36,6 +36,13 @@ public function setupOnce(): void
                 return $event;
             }
 
+            $transaction = SentrySdk::getCurrentHub()->getTransaction();
+            if ($transaction !== null) {
+                $event->setTransaction($transaction->getName());
+
+                return $event;
+            }
+
```

Comment:
We should likely move this after the hint was applied, as this should take precedence.